### PR TITLE
got something going with basic websockets

### PR DESCRIPTION
**Pull request by BordenJardine**
Also the README is now accurate again:
I agree with you that the express generator added too much stuff we don't need, so cannibalized the parts I liked and added them to your `server.js`

I never thought I would do homework on a _saturday_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # knit-works
-A project by Matthew Borden Jardine and Olivia Ruiz-Knott.
+A project by Matthew Borden Jardine and Olivia Heather Ramharacksing Ruiz-Knott.
```
Comment (oliviaruizknott) [inline README.md:2]:
```suggestion
A project by Matthew Borden Jardine and Olivia Bethany Ramharacksingh Ruiz-Knott.
```

so close buddy

File: server.js [hunk 2]
```diff
@@ -1,10 +1,46 @@
...
+app.use(express.static(path.join(__dirname, 'public')))
+
+// catch 404 and forward to error handler
+app.use(function(req, res, next) {
+  next(createError(404))
+})
 
 app.get('/', ( req, res ) => {
   res.send('Knitting is super cool! 🧶')
-});
```
Comment (oliviaruizknott) [inline server.js:6]: Are we going to be a no-semicolon project?—

File: server.js [hunk 3]
```diff
@@ -1,10 +1,46 @@
...
+  // render the error page
+  res.status(err.status || 500)
+  res.render('error')
+})
+
+startWebSockets(server)
+
+server.listen(port, () => {
+  console.log(`app listening on http://localhost:${port}`)
 });
```
Comment (oliviaruizknott) [inline server.js:44]: —or not?!

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<head>
+  <link rel="stylesheet" href="/stylesheets/style.css">
+  <script type="text/javascript" src="/javascripts/index.js"></script>
+</head>
+<body>
+  Knitting is Super Cool! 🧶
```
Comment (oliviaruizknott) [inline public/index.html:6]: Are you feeling good about the use of title case here?

**Review by oliviaruizknott — CHANGES_REQUESTED** [review]
Okay you should talk me through the `server.js` and `websockets.js` files tomorrow but in the meantime we have some very serious changes to be made.

File: README.md (hunk 1 above)
Comment (oliviaruizknott) [reply README.md:2]: You may merge once this requested change has been made.

Comment (BordenJardine) [reply README.md:2]: _fuck_

File: server.js (hunk 2 above)
Comment (BordenJardine) [reply server.js:6]: please please please

**Pull request merged**